### ISSUE TITLE
Fix the mis-detection of a missing accessible name when the element has a slot

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#1018]
- Markuplint version: `3.10.x`
- Parser lang: Svelte
- Node.js version: `18.x`
- OS: macOS

## Code Example

```svelte
<button>
  <slot />
</button>
```

## The current behavior

Require accessible name

## The expected behavior

No warning
